packages/react-composites/tests/browser/call/hermetic/fixture.ts

```diff
@@ -127,6 +127,51 @@ export function defaultMockRemoteParticipant(displayName: string): MockRemotePar
   };
 }
 
+/**
+ * Create the default {@link MockRemoteParticipantState} for a PSTN participant in a hermetic e2e test
+ *
+ * use to add PSTN participants to the {@link defaultCallAdapterState}
+ */
+export function defaultMockRemotePSTNParticipant(phoneNumber: string): MockRemoteParticipantState {
+  return {
+    identifier: { kind: 'phoneNumber', phoneNumber: `${phoneNumber}` },
+    state: 'Connecting',
```

Comment:
I'd set the "default" state to `Connected` and explicitly set the state in the test.

Why?

The test should be readable without jumping into the definition of any helper functions. The `state` is essential to understanding the test, so it should be set explicitly.

`default*` helper methods just provide a sensible value to use. Then explicitly set what matters so that the test code is both simple, and tells you exactly what is happening.

Reply:
will update this!